tests/TestCase/Http/RequestTransformerTest.php

```diff
@@ -259,7 +259,7 @@ public function testToCakeBaseSessionPath()
         $psr = ServerRequestFactory::fromGlobals($server);
         $cake = RequestTransformer::toCake($psr);
 
-        $this->assertEquals('/thisapp', ini_get('session.cookie_path'));
+        $this->assertEquals('/thisapp/', ini_get('session.cookie_path'));
```

Comment:
Could we reference that documentation somewhere? I can imagine someone could come across this, too, and could wonder about it.